src/DotLiquid/DotLiquid.csproj

```diff
@@ -35,6 +35,7 @@
   <ItemGroup>
     <PackageReference Include="SonarAnalyzer.CSharp" Version="5.11.0.1761" />
     <PackageReference Include="System.ValueTuple" Version="4.3.0" />
+	<Reference Include="System.Web" />
```

Comment:
The reference to `System.Web` bothers me, is it possible to do without it?

Reply:
Yes, I agree. I used `System.Net`